controller/api/destination/server.go

```diff
@@ -527,26 +514,6 @@ func podReceivingTraffic(pod *corev1.Pod) bool {
 	return !podTerminating && !podTerminated
 }
 
-// podToAddressSet converts a Pod spec into a set of Addresses.
-func podToAddressSet(k8sAPI *k8s.API, pod *corev1.Pod) *watcher.AddressSet {
```

Comment:
This function was used to make an AddressSet that we then just immediately indexed into with the PodID. It seems like a roundabout way to just make an Address—hence removing it.